server/server.go

```diff
@@ -471,15 +471,16 @@ func (s *session) start(ctx context.Context, wg *sync.WaitGroup) {
 	}()
 	for i := 0; ; i++ {
 		err := run(i)
-		if b, ok := err.(blamer); ok {
-			err = s.exclude(b.Blame())
-			if err != nil {
-				log.Printf("cannot continue after exclusion: %v", err)
-				return
-			}
-			continue
-		}
 		if err != nil {
+			var b blamer
```

Comment:
keep the control the same here as it was before. it should check for the blame error before the generic error.